GrandCentralBoard/Widgets/Harvest/HarvestWidgetSettings.swift

```diff
@@ -25,6 +26,7 @@ struct HarvestWidgetSettings: Decodable {
                                          clientID:        json => "clientID",
                                          clientSecret:    json => "clientSecret",
                                          numberOfDays:    json => "numberOfDays",
+                                         includeWeekends: json =>? "includeWeekends",
```

Comment:
Do you think it would be better to have optional Bool, or to check it like:
`includeWeekends = json =>? "includeWeekends" ?? false`

Cause I think, from the point of view of classes that are using the configuration, it would be better not to see optionals. Because then they are storing the logic about the default value.